packages/scrollable-image-ui/src/components/embedded-code-modal.js

```diff
@@ -1,8 +1,9 @@
-import orangutan from '@twreporter/orangutan'
 import PopoverHint from './simple-popover'
 import PropTypes from 'prop-types'
 import React, { useState } from 'react'
+import ScrollableImage from '@twreporter/scrollable-image/lib/build-code/index'
```

Comment:
For the time being, use `@twreporter/scrollable-image` rather than `@twreporter/orangutan` in this PR.

It will be updated to `@twreporter/orangutan` after this PR is merged and `scrollableImage` in `@twreporter/orangutan` is ready.

Reply:
`import ScrollableImage from '@twreporter/scrollable-image/lib/build-code/index'` 
-> 
`import { buildEmbeddedCode } from  '@twreporter/scrollable-image/lib/build-code/index'` 
?

Reply:
This was a mistake, thanks for the catch.

It has been fixed in commit `c2c33ba`.